app/controllers/foreman_register/hosts_controller.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+module ForemanRegister
+  class HostsController < ::ForemanRegister::ApplicationController
+    # Skip default filters
+    FILTERS = [
+      :require_login,
```

Comment:
```suggestion
      :require_login,
      :check_user_enabled,
```

After #7402 was merged, we need this here as well.

Reply:
#7402 got merged, but I plan to do some followup anyways. `:check_user_enabled` will do nada if User.current not set, so we don't need this.